### PR TITLE
ci: add stale bot

### PR DESCRIPTION
**Pull request by daixiang0**
# Description

Add stale bot to optimize issues/prs lifestyle.

### CODE REVIEW

**Comment by CodeMonkeyLeet** [comment]
LGTM. 

@pkedy this is a direct port of the merged https://github.com/dapr/dapr/pull/3248 PR to components-contrib. It will also require a maintainer to add the `pinned` issues label for the PR exception rule to be overridden as needed.

File: .github/workflows/dapr-bot-schedule.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-automerge
```
Comment (pkedy) [inline .github/workflows/dapr-bot-schedule.yml:6]: For consistency, should the name be `dapr-schedule`?

File: .github/workflows/dapr-bot-schedule.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        # open for it: https://github.com/actions/stale/issues/214
+        days-before-stale: 30
+        days-before-close: 7
+        stale-issue-message: >
+          This issue has been automatically marked as stale because it has not had activity in the
+          last 30 days. It will be closed in the next 7 days unless it is tagged (pinned, good first issue, help wanted or triaged/resolved) or other activity
+          occurs. Thank you for your contributions.
+        close-issue-message: >
+          This issue has been automatically closed because it has not had activity in the
+          last 37 days. If this issue is still valid, please ping a maintainer and ask them to label it as pinned, good first issue, help wanted or triaged/resolved.
```
Comment (pkedy) [inline .github/workflows/dapr-bot-schedule.yml:45]: The messages are slightly different than in dapr/dapr#3248. Not a big deal. Just calling it out.

**Review by pkedy — COMMENTED** [review]
Minor nit re: naming

File: .github/workflows/dapr-bot-schedule.yml (hunk 2 above)
Comment (daixiang0) [reply .github/workflows/dapr-bot-schedule.yml:45]: Since it has been updated by https://github.com/dapr/dapr/pull/3347.

**Comment by daixiang0** [comment]
@pkedy done.

**Comment by daixiang0** [comment]
ping @artursouza @yaron2

**Review by artursouza — APPROVED** [review]

**Pull request merged**